tutorial-week-5.md

```diff
@@ -41,14 +41,17 @@ Changes not staged for commit:
 ```
 4.) Now you can simply `git add`, `git commit`, and `git push`.
   
+###Extras:
```

Comment:
It would be cool to include a reason _why_ you might want to `git stash`...why wouldn't you want to commit files? Etc :)